pkg/logging/utils.go

```diff
@@ -0,0 +1,102 @@
+package logging
+
+import (
+	"regexp"
+	"strings"
+)
+
+// MaskSSN finds and masks any SSN in the format YYYYMMDDXXXX in the input string
+func MaskSSN(input string) string {
```

Comment:
dentech-floss is placement for open source libs
I suggest clarifying that this is specifically about the Swedish SSN.

Reply:
That is a very good point! 👍 Will fix

Reply:
But then again the maskPhone currently primarily handles Swedish phone nbrs too. Maybe this does not belong in the floss-logging. 🤔 ...will reconsider this.

Reply:
After marinating on this over the weekend I think maybeit's better for now at least to keep this in the service instead. Then maybe later we can refine it be general enough to belong here.

Will close this PR.